barcode_detection/decoding/zxing_method/docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9
+
+WORKDIR /workspace
+RUN python3 -m pip install zxing
```

Comment:
Let's add `requirements.txt` file (with versions!) to this folder, then you can write `pip install -r requirements.txt`